templates/index.html

```diff
@@ -117,5 +121,6 @@ <h3 class="mb1">
     {{> ampstart/footer.snip.html}}
     {{> analytics.html}}
     {{> serviceworker.html}}
+    {{> consent.html}}
```

Comment:
There are a few more templates where this needs to be injected: 

* preview-*.html
* example.html

Ideally, could you create a new template `footer.html` which then includes analytics.html, service worker.html and consent.html

Reply:
Makes sense!

The site already has a `footer.snip.html` file, though.  Is there a logical name for another footer template?  Or can I simply include the three templates you mention in `footer.snip.html` ?

Reply:
That's for the ampstart template which is not used on all pages (e.g. the previews). 

How about `footer-boilerplate.html`?

Reply:
Ok - done!
The only part I'm concerned about is preview.html, where analytics and serviceworker were only included this way:

```
    {{^document.isAmpStory}}
    {{> analytics.html}}
    {{> serviceworker.html}}
    {{/document.isAmpStory}}
```

is that ok?

Reply:
You can't use it there - you have to inject it manually similar to analytics.

Reply:
Ok!

What pages use the preview* templates?  I'll test.